.bazelci/presubmit.yml

```diff
@@ -41,3 +43,25 @@ platforms:
       - "-//tests:pip_repository_entry_points_example"
     test_flags:
       - "--test_tag_filters=-fix-windows"
+  rbe_ubuntu1604:
+    shell_commands:
+      - sed -i 's/^# load("@bazelci_rules/load("@bazelci_rules/' WORKSPACE
```

Comment:
I don't see what these match in WORKSPACE, is this no longer needed?

Reply:
Appears not, good catch!